osu.Game/Screens/Select/FooterV2/BeatmapOptionsPopover.cs

```diff
@@ -0,0 +1,148 @@
+            {
+                AutoSizeAxes = Axes.Both,
+                Direction = FillDirection.Vertical,
+                Spacing = new Vector2(3),
+            };
+
+            addButton(@"Manage collections", FontAwesome.Solid.Book, () => manageCollectionsDialog?.Show());
+            addButton(@"Delete all difficulties", FontAwesome.Solid.Trash, () => songSelect?.DeleteBeatmap(), colours.Red);
+            addButton(@"Remove from unplayed", FontAwesome.Regular.TimesCircle, null);
+            addButton(@"Clear local scores", FontAwesome.Solid.Eraser, () => songSelect?.ClearScores());
+
+            if (songSelect != null && songSelect.AllowEditing)
+                addButton(@"Edit beatmap", FontAwesome.Solid.PencilAlt, () => songSelect.Edit());
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            ScheduleAfterChildren(() => GetContainingInputManager().ChangeFocus(this));
+        }
+
+        private void addButton(LocalisableString text, IconUsage icon, Action? action, Color4? colour = null)
+        {
+            var button = new OptionButton
+            {
+                Text = text,
+                Icon = icon,
+                TextColour = colour,
+                Action = () =>
+                {
+                    Hide();
+                    action?.Invoke();
+                },
+            };
+
+            buttonFlow.Add(button);
+        }
+
+        private partial class OptionButton : OsuButton
+        {
+            public IconUsage Icon { get; init; }
+            public Color4? TextColour { get; init; }
+
+            public OptionButton()
+            {
+                Size = new Vector2(265, 50);
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OverlayColourProvider colourProvider)
+            {
+                BackgroundColour = colourProvider.Background3;
+
+                SpriteText.Colour = TextColour ?? Color4.White;
+                Content.CornerRadius = 10;
+
+                Add(new SpriteIcon
+                {
+                    Blending = BlendingParameters.Additive,
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    Size = new Vector2(17),
+                    X = 15,
+                    Icon = Icon,
+                    Colour = TextColour ?? Color4.White,
+                });
+            }
+
+            protected override SpriteText CreateText() => new OsuSpriteText
+            {
+                Depth = -1,
+                Origin = Anchor.CentreLeft,
+                Anchor = Anchor.CentreLeft,
+                X = 40
+            };
+        }
+
+        protected override bool OnKeyDown(KeyDownEvent e)
+        {
+            // don't absorb control as ToolbarRulesetSelector uses control + number to navigate
+            if (e.ControlPressed) return false;
```

Comment:
This should have test coverage somewhere. Eventually.

Reply:
The old overlay had it in screen navigation:
https://github.com/ppy/osu/blob/491f15e68ced2be97736445775c62eeccfae60ca/osu.Game.Tests/Visual/Navigation/TestSceneScreenNavigation.cs#L463

Reply:
Probably fine to ignore this until the swap happens then I guess.